ziggurat-core-metrics/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "ziggurat-core-metrics"
```

Comment:
zc-metrics maybe?

Reply:
We should discuss the naming convention and make some guidelines commited to the repo.